script/shared/utils.ts

```diff
@@ -20,7 +20,7 @@ export function askQuestion(query: string) {
   return new Promise((resolve) =>
     rl.question(`${query} [Y/n] `, (ans) => {
       rl.close();
-      resolve(ans.toLowerCase() == 'y' ? true : false);
+      resolve(ans.toLowerCase() == 'y' || ans == '' ? true : false);
```

Comment:
We intended this because we believed it should not be executed without explicit agreement. However, considering 'enter' to be equivalent to 'y' seems dangerous. What do you think?

Reply:
This convention is widely used in many CLI tools, making it familiar to users. See [this](https://askubuntu.com/questions/322600/do-you-want-to-continuey-n-why-the-upper-case) for more info.

- It helps users quickly understand what will happen if they don't provide any input.
- The uppercase letter stands out, making it easier for users to see the default option at a glance.

In this case, the prompt already has the uppercase Y (line 21), this PR just fixes the implemented behavior.

Reply:
Just to clarify - I understand the original intention behind requiring explicit agreement. However, I think considering 'enter' to be equivalent to 'y' is a common convention in many CLI tools. It allows for quicker interaction, especially when the default option is the most likely choice. If we stick to requiring explicit input, we should update the prompt to remove the suggestion that 'y' is the default. For example: 
```
Do you want to proceed? (y/n)
```
Alternatively, we could add a clarifying message after the user presses enter without inputting a choice, such as: "Please enter 'y' to proceed or 'n' to abort." This would help avoid any confusion.

Let me know your thoughts, and we can decide on the best approach.

Reply:
Oh okay, it's conventional thing. then it's okay to merge it!